src/general/OptionsMenu.cs

```diff
@@ -1114,4 +1114,10 @@ private void UpdateSelectedLanguage(Settings settings)
             languageSelection.Selected = languages.IndexOf(settings.SelectedLanguage.Value);
         }
     }
+
+    private void OnLogButtonPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        OS.ShellOpen(ProjectSettings.GlobalizePath("user://" + Constants.LOGS_FOLDER_NAME));
```

Comment:
As said in another comment I think LOGS_FOLDER_NAME should be renamed to LOGS_FOLDER, which would include the "user://" part in it.

Reply:
Actually that might not be possible as the startup actions actually needs the logs folder without the user:// prefix. So I think if this code works, that would be nice:
```
public const string LOGS_FOLDER_NAME = "logs";
public const string LOGS_FOLDER = "user://" + LOGS_FOLDER_NAME;
```
If that doesn't then leaving this as is, is fine.

Reply:
ok just changed that so

Reply:
I try and I tell  you

Reply:
ok  it's working I commit it so